installing/installing_ibm_cloud_public/user-managed-encryption-ibm-cloud.adoc

```diff
@@ -0,0 +1,42 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="user-managed-encryption-ibm-cloud"]
+= User-managed encryption for {ibm-cloud-title}
+include::_attributes/common-attributes.adoc[]
+:context: user-managed-encryption-ibm-cloud
+
+toc::[]
+
+By default, provider-managed encryption is used to secure the following when you deploy an {product-title} cluster:
+
+* The root (boot) volume of control plane and compute machines.
+* Persistent volumes (data volumes) that are provisioned after the cluster is deployed.
+
+You can override the default behavior by specifying an {ibm-name} Key Protect for {ibm-cloud-name} (Key Protect) or {ibm-name} Hyper Protect Crypto Services (Hyper Protect Crypto Services) root key as part of the installation process.
+
+When you bring our own root key, you modify the installation configuration file (`install-config.yaml`) to specify the CRN of the root key by using the `encryptionKey` parameter.
+
+You can specify that:
+
+* The same root key be used be used for all cluster machines. You do so by specifying the key as part of the cluster's default machine configuration.
++
+When specified as part of the default machine configuration, all managed storage classes are updated with this key. As such, data volumes that are provisioned after the installation are also encrypted using this key.
+
+* Separate root keys be used for the control plane and compute machine pools.
```

Comment:
```suggestion
* Separate root keys may be used for the control plane and compute machine pools.
```

Reply:
nit:  may/can, whatever sounds best.

Reply:
In the context of the paragraph, this is grammatically correct. You can specify that separate root keys be used...

Reply:
Oh I see. Gotcha, missed that part.